saleor/graphql/attribute/tests/queries/test_attribute_query.py

```diff
@@ -453,7 +453,9 @@ def test_attributes_in_collection_query(
     other_collection = Collection.objects.create(
         name="Other Collection",
         slug="other-collection",
-        description="Description",
+        description={
+            "blocks": [{"data": {"text": "Test description."}, "type": "paragraph"}]
```

Comment:
Maybe we should have a function that would turn some text into simply editor.js content? Something like `dummy_editorjs("Test description")` that would output the above structure? Just for tests.